docs/maps/maps-aggregations.asciidoc

```diff
@@ -1,8 +1,51 @@
 [role="xpack"]
+[[maps-aggregations]]
+== Plot big data without plotting too much data
+
+The Maps application uses {ref}/search-aggregations.html[aggregations] to plot large data sets without overwhemling your network or your browser.
+
+Aggregations group your documents into buckets and calculate metrics for each bucket.
+Your documents stay in Elasticsearch and only the metrics for each group are returned to your computer.
+
+
+[float]
+[[maps-grid-aggregation]]
+=== Grid aggregation
+
+The *Grid aggregation* source uses {ref}/search-aggregations-bucket-geotilegrid-aggregation.html[GeoTile grid aggregation] to group your documents into grids. You can calculate metrics for each gridded cell.
+
+You can symbolize grid aggregation metrics as:
+
+*Points*:: Creates a <<vector-layer, vector layer>> with a point for each gridded cell.
+The point location is the weighted centroid for all geo-points in the gridded cell.
+
+*Grid rectangles*:: Creates a <<vector-layer, vector layer>> with a bounding box polygon for each gridded cell.
+
+*Heat map*:: Creates a <<heatmap-layer, heat map layer>> that clusters the weighted centroids for each gridded cell.
+
+
+[float]
+[[maps-top-hits-aggregation]]
+=== Most recent entities
+
+Most recent entities uses {es} {ref}/search-aggregations-bucket-terms-aggregation.html[terms aggregation] to group your documents by entity.
+Then, {ref}/search-aggregations-metrics-top-hits-aggregation.html[top hits metric aggregation] accumulates the most recent documents for each entry.
+
+Set up most recent entities on a *Documents* source by enabling "Show most recent documents by entity"
```

Comment:
"on a Documents source" sounds awkward.  Can it be "Document source" or "Document's source"?

Reply:
How about 

```
Most recent entities is available for vector layers with *Documents* source. Set up most recent entities by enabling "Show most recent documents by entity"
```

Reply:
That works.  Be sure to add the ending punctuation.